src/components/chart/components/range-selection.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react'
+
+import { Flex, Text } from '@chakra-ui/react'
+
+import { colors } from '@/lib/styles/colors'
+
+type RangeSelectionItemProps = {
+  label: string
+  isSelected: boolean
+  onClick: () => void
+}
+
+function RangeSelectionItem({
+  label,
+  onClick,
+  isSelected,
+}: RangeSelectionItemProps) {
+  const color = isSelected ? '#1C2929' : colors.icGray5
+  const backgroundColor = isSelected ? '#F2F2F2' : 'none'
```

Comment:
similar questions here RE: palette

Reply:
Updated colors according to what's available in interfaces. `#F2F2F2` wasn't available so leaving it as custom for now.